app/src/main/java/com/android/streetworkapp/ui/utils/CustomDialog.kt

```diff
@@ -85,9 +97,20 @@ fun CustomDialog(
 
 enum class DialogType {
   QUERY, // Have submit / cancel buttons + use OnSubmit / onDismiss.
-  INFO; // Only display the content + use onDismiss.
+  INFO, // Only display the content + use onDismiss.
+  CONFIRM;
```

Comment:
It would be a good idea to add a comment like for the enums above.

Reply:
Thanks again for catching this, I forgot to change it afterwards !